src/sidebar/services/frame-sync.js

```diff
@@ -37,21 +37,26 @@ export function formatAnnot({ $tag, target, uri }) {
 /**
  * Return the frame which best matches an annotation.
  *
- * If there is a frame whose URL exactly matches the annotation, it will be
- * returned. Otherwise the main frame will be returned. Currently this may fail
- * to return the expected frame if an annotation was fetched from h that
- * does not match one of the frame URIs. This can happen due to URI expansion /
- * document equivalence in h.
```

Comment:
I moved the comments to be above the relevant implementation lines, so they are less likely to get out of date. Also this is an internal function so it isn't really important for this to be "public" documentation.